vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -69,397 +70,406 @@
 
 #include <tuple>
 
-namespace react { namespace uwp {
+namespace react {
+  namespace uwp {
```

Comment:
You need to manually fix the format problem. VS always messed it up.
You should remove the first 4 spaces everywhere below